feat: Localize revamped modlist

0neGal opened the pull request:
This PR is for merging localization of the revamped modlist. Replacing the list based one.

#### New localization keys

```json
"gui.mods.remove": "Remove",
"gui.mods.unknown_author": "Unknown",
```

#### Maintainers

 - [ ] German - @DxsSucuk
 - [ ] French - @Alystrasz
 - [ ] Spanish - @AA-Delta

DxsSucuk commented:
I sadly do not have the ability to commit the translations for the next few hours. I can provide them per comment if needed thou.

0neGal commented:
Off topic, but on top of the localization, if somebody on Windows is able to run and verify everything is still working, that'd be very nice! As there's been major changes since the last release all over the codebase. Possibly breaking stuff...

0neGal commented:
> I sadly do not have the ability to commit the translations for the next few hours. I can provide them per comment if needed thou.

They're not required this very moment, I still have a few extra things to fix before a release, but the language strings should be finalized...

DxsSucuk commented:
> Off topic, but on top of the localization, if somebody on Windows is able to run and verify everything is still working, that'd be very nice! As there's been major changes since the last release all over the codebase. Possibly breaking stuff...

Could do it in like 2-3 hours. Can you provide a dev build or do I need to built it locally?

0neGal commented:
> > Off topic, but on top of the localization, if somebody on Windows is able to run and verify everything is still working, that'd be very nice! As there's been major changes since the last release all over the codebase. Possibly breaking stuff...
> 
> Could do it in like 2-3 hours. Can you provide a dev build or do I need to built it locally?

You can get it from the automatic GitHub action, [here's the current dev Windows build](https://github.com/0neGal/viper/suites/10717216875/artifacts/537065805). Keep in mind it's a zip file with the setup and portable executable, so you'll have to extract it first.

Alystrasz commented:
> Off topic, but on top of the localization, if somebody on Windows is able to run and verify everything is still working, that'd be very nice! As there's been major changes since the last release all over the codebase. Possibly breaking stuff...

Everything seems to be running just fine on Windows 10.

DxsSucuk commented:
Couldn't find more then this 
![image](https://user-images.githubusercontent.com/53257574/216186039-7da56808-a130-47c9-84fc-6c3d842fd606.png)

Everything else looks to be fine.

0neGal commented:
> Everything seems to be running just fine on Windows 10.

Lovely!

> Couldn't find more then this

That is actually the next bug I wanted to fix :p
